packages/calcite-components/src/components/flow/flow.scss

```diff
@@ -1,61 +1,78 @@
+/**
+ * CSS Custom Properties
+ *
+ * These properties can be overridden
+ *
+ * @prop --calcite-internal-flow-animation-timing: Specifies the animation timing for the flow.
```

Comment:
I think these need to go in a separate "internal block" so they aren't documented (or, just remove the `@prop`) - cc @alisonailea for format advice here as it will apply to all other PR